docs/client/README.md

```diff
@@ -0,0 +1,15 @@
+# Semantic Conventions For Client SDKs
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines general semantic conventions for client-side applications.
+The most common types of client applications are web applications (that run in a web browser)
+and mobile apps (that run on a phone or tablet).
```

Comment:
I think it would make sense to also call out some of the non-common types (IoT? Desktop applications??)